submissions/olabodeAmusan/Database/init2.sql

```diff
@@ -0,0 +1,34 @@
+CREATE DATABASE transportation_company;
+USE transportation_company;
+CREATE TABLE passengers(
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(250) NOT NULL,
+sex ENUM('male','female') NOT NULL DEFAULT 'male',
+age INT NOT NULL DEFAULT '0',
+PRIMARY KEY(id)
+);
+
+CREATE TABLE trips(
+id INT NOT NULL AUTO_INCREMENT,
+passenger_id INT NOT NULL,
+passenger_class ENUM('1','2','3') NOT NULL DEFAULT '1',
+ticket_number VARCHAR(8) NOT NULL,
+trip_fare DECIMAL(10,2) NOT NULL,
+assigned_cabin VARCHAR(5) NOT NULL DEFAULT 'nil',
+parent INT NOT NULL,
+siblings INT NOT NULL,
```

Comment:
Try to give lines 18 and 20 descriptive names to reflect the description given in the task.